devices/Chsc6540/samples/Program.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Threading;
+using Iot.Device.Chsc6540;
+using nanoFramework.Hardware.Esp32;
+
+//////////////////////////////////////////////////////////////////////
+// when connecting to an ESP32 device, need to configure the I2C GPIOs
+// used for the bus
+Configuration.SetPinFunction(21, DeviceFunction.I2C1_DATA);
+Configuration.SetPinFunction(22, DeviceFunction.I2C1_CLOCK);
+
+I2cConnectionSettings settings = new(1, Chsc6540.DefaultI2cAddress);
+using I2cDevice device = I2cDevice.Create(settings);
+using GpioController gpio = new();
+
+using Chsc6540 sensor = new(device);
+sensor.SetInterruptMode(true);
+
+var touchInt = gpio.OpenPin(39, PinMode.Input);
+touchInt.ValueChanged += TouchInt_ValueChanged;
+
+// This will enable an event on GPIO39 on falling edge when the screen if touched
+//gpio.RegisterCallbackForPinValueChangedEvent(39, PinEventTypes.Falling, TouchInterrupCallback);
+
+while (true)
+{
+    Thread.Sleep(20);
+}
+
+
+
+void TouchInt_ValueChanged(object sender, PinValueChangedEventArgs e)
+{
+    Debug.WriteLine("Touch interrupt");
+}
+
```

Comment:
Can you add a little comment on this code and how to use it? Or if not relevant, just remove it